detections/process_ja3_ja3s_map.yml

```diff
@@ -0,0 +1,79 @@
+asset_type: Network
+confidence: low
+creation_date: '2019-10-25'
+data_metadata:
+  data_source:
+    - Carbon Black, Bro
+  data_sourcetypes:
+    - cb:response, bro:ssl
+  providing_technologies:
+    - Carbon Black Response, Bro
+description: This search will map Carbon Black Reponse process data to Bro SSL network data w/JA3 signatures, as well as the JA3s to fingerprint server response.
+detect: 
+  splunk:
+    correlation_rule:
+      notable:
+        nes_fields: dest, user, process
+        rule_description: This search will map Carbon Black Reponse process data to Bro SSL network data w/JA3 and J3S signatures.
+        rule_title: Process to JA3+JA3s Signature
+      risk:
+        risk_object: dest
+        risk_object_type:
+          - system
+        risk_score: 10
+      schedule:
+        cron_schedule: 0 * * * *
+        earliest_time: -70m@m
+        latest_time: -10m@m
+      search: ( index=carbonblack netconn ) OR ( index=bro ja3 )
```

Comment:
We probably want to remove the index and instead datamodel or sourcetypes in the queries

We need the ja3_lookup file if we can. Could you give it that?

Reply:
can you lmk what you want w.r.t. source types? Since we didn't build it off the data model I'm not too sure what you guys want there.

Unfortunately, we're unable to share the lookup file (ja3-ja3s_dict.csv, but it's a CSV with one column and the following format:
ja3_lookup
<ja3:ja3s>

Reply:
sorry for the delay on this, we are working on improving the process/PRs created by community for security-content repo. Stay tuned! 

No worries on the lookup files. thanks again!